ariadne/cardano/src/Ariadne/Config/CLI.hs

```diff
@@ -238,6 +240,13 @@ cliWalletParser = do
      , metavar "BYTE"
      , help "Entropy size in bytes, valid values are: [16, 20, 24, 28, 32]"
      ]
+  cli_wcAcidDBPath <- fmap Just <$> strOption $ mconcat 
```

Comment:
What's the purpose of making `cli_wcAcidDBPath` `Maybe` if it's always `Just`?
And why did you make it always `Just`? Shouldn't it be `optional`?